my_enumerable.rb

```diff
@@ -0,0 +1,21 @@
+# This module provides custom enumerable methods.
+module MyEnumerable
+  def all?
+    arr = []
+    @list.each { |i| arr.push(i) if yield i }
+    puts arr.length == @list.length
+  end
+
+  def any?
+    @list.each do |i|
+      puts true if yield i
+    end
+    puts false
+  end
```

Comment:
- [ ] When i test your any? method as shown in the image below, it returns two answers `true & false` whereas it should only return `true`, please kindly look at your implementation and make the necessary corrections so that your code works properly :+1: :sparkles: